    @Test
    public void testUpdateBlobWithBlob() throws SQLException {
        try (Database db = db()) {
        Blob blob = new JDBCBlobFile(new File("src/test/resources/big.txt"));
        insertPersonBlob(db);
        db //
                .update("update person_blob set document = :doc") //
                .parameter("doc", blob) //
                .counts() //
                .test() //
                .awaitDone(TIMEOUT_SECONDS, TimeUnit.SECONDS) //
                .assertValue(1) //
                .assertComplete();
        }
    }